plugins/downloader/utils.js

```diff
@@ -1,4 +1,4 @@
-const downloadsFolder = require("downloads-folder");
+const { app } = require("electron");
 
-module.exports.getFolder = (customFolder) => customFolder || downloadsFolder();
+module.exports.getFolder = (customFolder) => customFolder || app.getPath("downloads");
```

Comment:
The `app` module from electron is only available in the [main process](https://github.com/electron/electron/blob/master/docs/api/app.md#app), not the renderer one, so it breaks the downloader - a [fix](https://stackoverflow.com/questions/47957479/how-to-use-electrons-app-getpath-to-store-data) for that is to use `(electron.app || electron.remote.app).getPath("downloads")`

Reply:
Woups I missed that, good catch 🙂